src/nvim/version.c

```diff
@@ -18,10 +18,23 @@
 #include "nvim/misc2.h"
 #include "nvim/screen.h"
 #include "nvim/strings.h"
-#include "nvim/version_defs.h"
+
+// version info generated by the build system
+#include "auto/versiondef.h"
+
+#define STR_(x) #x
+#define STR(x) STR_(x)
+
+// for the startup-screen
```

Comment:
I think more people know it as "the startup-screen" but I will append `:intro` as well.